packages/protocol/contracts/common/Accounts.sol

```diff
@@ -396,15 +596,39 @@ contract Accounts is
     }
   }
 
+  /**
+   * @notice Returns the signer for the specified account and role. If no 
+   * signer has been specified it will return the account itself.
+   * @param _account The address of the account.
+   * @param role The role of the signer.
+   */
+  function getDefaultSigner(address _account, bytes32 role) public view returns (address) {
```

Comment:
I don't know if this got lost/already addressed in the comments, but wdyt about separating out a function that does the legacy checks and one that doesn't? I.e. if I'm using a non-legacy role, I know that I don't have to check for these and save gas that way